derive_client/cli/_account.py

```diff
@@ -0,0 +1,62 @@
+"""CLI commands related to wallet."""
+
+from __future__ import annotations
+
+import rich_click as click
+
+from ._columns import COLLATERAL_COLUMNS, ORDER_COLUMNS, POSITION_COLUMNS, SUBACCOUNT_COLUMNS
+from ._utils import explode_struct_column, struct_to_series, structs_to_dataframe
+
+
+@click.group("account")
+@click.pass_context
+def account(ctx):
+    """Account details."""
+
+
+@account.command("get")
+@click.pass_context
+def get(ctx):
+    """Account details."""
+
+    client = ctx.obj["client"]
+    account = client.account.get()
+    series = struct_to_series(account)
+
+    print("\n=== Account Info ===")
+    print(series.drop("fee_info"))
+
+    print("\n=== Fee Info ===")
+    print(struct_to_series(series.fee_info))
+
+
+@account.command("portfolios")
+@click.pass_context
+def portfolios(ctx):
+    """Get all portfolios of a wallet."""
+
+    client = ctx.obj["client"]
+    portfolios = client.account.get_all_portfolios()
+
```

Comment:
This is effectively the `.get_subaccounts`, except the latter just returns subaccount ids, whereas this returns their state (hence I've omitted "get_subaccounts". Also the `drv account get` returns account state, including subaccount ids.